PokeAlarm/Filters/MonFilter.py

```diff
@@ -171,6 +171,16 @@ def __init__(self, mgr, name, data):
         self.weather_ids = self.evaluate_attribute(
             event_attribute='weather_id', eval_func=operator.contains,
             limit=BaseFilter.parse_as_set(get_weather_id, 'weather', data))
+        self.boosted_weather_ids = self.evaluate_attribute(
+            event_attribute='boosted_weather_id', eval_func=operator.contains,
+            limit=BaseFilter.parse_as_set(
+                get_weather_id, 'boosted_weather', data))
+        self.boosted_weather = self.evaluate_attribute(
+            event_attribute='boosted_weather_id',
+            eval_func=lambda d, v:
+            (d is True and v != 0 and v is not None)
```

Comment:
nit: this would be cleaner if you define the function somewhere else and just pass it in here